frontend/device/generic/device.lua

```diff
@@ -405,9 +405,10 @@ end
 
 function Device:install()
     local ConfirmBox = require("ui/widget/confirmbox")
+    local ignore_events = {"swipe", "hold", "hold_release", "hold_pan", "touch", "pan", "pan_release"}
```

Comment:
If we really need to have some ConfirmBox and InfoMessage unmovable (and less need to have some events ignored but other not ignored), it could be simpler to pass `unmovable=true` to these widgets, which would pass it to their MovableContainer, and have:
```lua
function MovableContainer:init()
    if Device:isTouchDevice() and not self.unmovable then
```
(Even if it is strange to have an unmovable MovableContainer :) but that can be explained with a comment `-- this can be passed whe needed to have MovableContainer present but a no-op, so we don't have to change the widget layout` or something.)